# External Reference Data Catalog

A catolog of external references available in the impala database.  

This README provides a summary of available resources. Click on a link to view more information about each database, including: 
<ul>
<li>Database information</li>
<li>Database definition</li>
<li>Sample queries</li>
<li>Source information</li>
<li>Mapping to related resources</li>
</ul>

##Gene annotation resources
Annotate variants with gene definitions based upon genomic location. Use these files to find exact matches with a variant based on chromosome, start, stop, reference and alternate alleles. 

Uniprot annotations can be used for mapping variants to PFAM database. 
<ul>
<li><a href="https://github.com/summerela/external-reference-data-catalog/blob/master/UCSC_genedefs.md", target="_blank">UCSC gene definitions</a></li>
<li><a href="https://github.com/summerela/external-reference-data-catalog/blob/master/uniprot_genedefs.md", target="_blank">Uniprot gene definitions</a></li>
<li><a href="https://github.com/summerela/external-reference-data-catalog/blob/master/refseq_genedefs.md", target="_blank">RefSeq gene definitions</a></li>
<li><a href="https://github.com/summerela/external-reference-data-catalog/blob/master/ensembl_genedefs.md", target="_blank">Ensembl gene definitions</a></li>
</ul>

##Functional annotation resources
Functional annotation resources can be used to find overlap between a variant and a region in one of the following resources. <ul>
<li>dbSNP: Locate single base nucleotide subsitutions and short deletion and insertion polymorphisms</li>
<li>phastCons: Conserved Elements</li>
<li>Transfac Matrix Database: Transcription factor binding sites</li>
<li>Cytobands: Cytogenic band identification</li>
<li>miRBase: Variants disrupting microRNA</li>
<li>snoRNABase: Variants disrputing snoRNA</li>
<li>TargetScanS: find variants disrupting predicted microRNA binding sites</li>
<li>GenomicSuperDups: find variants located in segmental duplications</li>
<li>Database of Genomic Variants: located previously reported structural variants</li>
<li>GWASCatalog: find variants reported in previously published GWAS</li>
<li>ENCODE: identify variants that disrupt promoter, enhancer, elongation and repressor regions</li>
</ul>

##Comparison resource

